### ISSUE TITLE
Disabling Desktop mode blocks LGS settings

### ISSUE THREAD

**Issue by Ulym38** [antonpup/Aurora#725]
I'm using Aurora 0.6.1-dev2 with a Logitech G410 keyboard. When I disable desktop mode in Aurora, my LGS settings doesn't show up on my keyboard (The keyboard stays black).
So, I decide to put the "Show default lightning(CUE, LGS, etc) when Desktop is disabled" option on.

However, when I enable this option, Aurora custom profiles don't work anymore.
I tried this on Skyrim (custom made) and Borderlands 2 (integrated).

The only way I found to fix this issue is to disable this option and go in the LGS lightning settings.

Is there a way to prevent Aurora from blocking LGS settings when Desktop mode is disabled without blocking other custom profiles ?

Thank you in advance for your answer.

**Comment by simon-wh** [comment]
I just tried to replicate your issue using 0.6.1-dev2 and the current dev branch and couldn't replicate it on either using Borderlands 2 and Spotify as a custom profile. Could you provide your latest log file on the day where you've replicated this issue? I have a feeling it may be an issue with re-initializing Logitech integration.

**Comment by Ulym38** [comment]
Here is my log file : https://pastebin.com/zTtSc14J
To simplify the issue, the problem I found is : when I disable desktop mode in Aurora, my keyboard doesn't use LGS default profile.
And the "Show default lightning(CUE, LGS, etc) when Desktop is disabled" option disables every Aurora custom profile.

**Comment by simon-wh** [comment]
From your log, it shows that Logitech integration is being disabled and enabled properly, but for some reason, it isn't working. With "custom profiles don't work" what do you see on your keyboard and in Aurora (UI and the Debug window)?

**Comment by Ulym38** [comment]
I noticed that the issue occured because I was using "Foreground app scan" instead of the default "Windows event" in the "Application Detection Method" setting...
When I switch back to the default option, Aurora works perfectly fine.

I switched to this mode because Borderlands 2 profile was not reading life and shield data in a multiplayer game, however, it caused more problems than expected.

To answer to your question, lights appeared on the keyboard when I edited the profile but they didn't when I launched the game.

I'm sorry I caused you some worries on this issue, it's just due to my carelessness.

Keep up the good work !
